docs/content/Authentication-&-Authorization.md

```diff
@@ -162,9 +161,9 @@ a token with this json structure:
 ```
 
 In this case, the `{ "user_id": 42 }` object will be accessible as
-[USER_CONTEXT][link-user-context] in the Cube.js Data Schema.
+[SECURITY_CONTEXT][link-security-context] in the Cube.js Data Schema.
 
-[link-user-context]: /cube#context-variables-user-context
+[link-user-context]: /cube#context-variables-security-context
```

Comment:
I think this reference is duplicated, should be safe to remove

Reply:
👍